### PR TITLE
Update `index.json` by `npm run generate-list`

### PR DESCRIPTION
**Pull request by ybiquitous**
This change also fixes `generate-list.js` to keep functions that end with `X|Y|Z`, like `rotatex` or `rotatey` (`rotateX` or `rotateY`).

### CODE REVIEW

File: generate-list.js [hunk 1]
```diff
@@ -27,7 +27,19 @@ const references = [
 			)
 				.map((node) => node.textContent || '')
 				.map((name) => name.replace('()', ''));
-			return list;
+
+			const additinal = [];
```
Comment (Mouvedia) [inline generate-list.js:31]: typo

You meant additional?

Comment (ybiquitous) [reply generate-list.js]: Good catch! Fixed via 8c073f8.

**Comment by niksy** [comment]
Thank you! Available in [3.2.0](https://github.com/niksy/css-functions-list/releases/tag/v3.2.0).

**Comment by ybiquitous** [comment]
Thank you for your quick release! 🎉

**Pull request merged**